Makefile

```diff
@@ -53,7 +53,7 @@ tmp/dev_image_id: Dockerfile.dev scripts/devtools.sh
 build: build/dev
 
 .PHONY: build/dev
-build/dev: test *.go
+build/dev: *.go
```

Comment:
this accidental? pls revert

Reply:
will revert for now; but I was finding it hindered rapid development. I do want to test - but the slow test targets in the Makefile shouldn't be a pre-req for a quick development build.